packages/protocol/contracts/L1/libs/LibVerifying.sol

```diff
@@ -88,6 +88,10 @@ library LibVerifying {
                 fc.prover = address(0);
                 fc.provenAt = 0;
             }
+
+            unchecked {
+                i++;
```

Comment:
Pre-increment is cheaper. (Ran 50+ tests for average gas consumption calculations to verify.)